release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -321,13 +321,26 @@ With this release, the etcd tuning parameters can be set to values that optimize
 
 With this release, `dasd` image artifacts for the `s390x` architecture are removed from the {product-title} image building pipeline. You can still use the `metal4k` image artifact, which is identical and contains the same functionality.
 
+[discrete]
```

Comment:
Noticed during rebase that this was missed in a recent unrelated merge, adding it now.